coffeecard/CoffeeCard/Helpers/MobilePay/ResponseMessage/IMobilePayAPIResponse.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace CoffeeCard.Helpers.MobilePay.ResponseMessage
+{
+    public interface IMobilePayAPIResponse
+    {
```

Comment:
Any reason TransactionId is not in the interface? It seems to be a field that all response messages shares

Reply:
We cant have fields in interfaces, so we could consider having some inheritance with a super classes or abstract class but personal opinion is when we have to model the objects the MobilePay API is responding with better just keep them exactly as they are.

But I get that the need of an  empty interface is a little weird since it has methods which mus implemented. Motivation for the interface is only to constraint that response messages must be of this type

Reply:
Since you are using getters and setters for it you should be able to add those if we wanted to. I do get your reasoning though, so maybe it's better left as is. Up to you.